foundation/org.eclipse.persistence.core/src/org/eclipse/persistence/internal/databaseaccess/DatabasePlatform.java

```diff
@@ -2475,8 +2485,8 @@ public Object executeStoredProcedure(DatabaseCall dbCall, PreparedStatement stat
         ResultSet resultSet = null;
         if (!dbCall.getReturnsResultSet()) {// no result set is expected
             if (dbCall.isCursorOutputProcedure()) {
-                result = accessor.executeNoSelect(dbCall, statement, session);
```

Comment:
woops, shouldn't have deleted this line! Must fix this in another PR. I guess this want caught sooner since Oracle testing was being done on master mostly.